DataHub/UserInterface/01a_Attributes.md

```diff
@@ -112,33 +109,36 @@ The list displays all attributes. Depending on the settings, the displayed colum
 - [SAVE]   
     Click this button to save the attribute. The *Create attribute* view is closed.
 
-**BASIC DATA**
-
 - *Name (Language)*   
     Enter an attribute name in the selected language.
 
 - *Description (Language)*   
     Enter a description to the attribute in the selected language.
 
 - *Data type*   
-    Click the drop-down list and select an attribute data type. The ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned sets* field is unlocked.   
+    Click the drop-down list and select an attribute data type. The ![Add](../../Assets/Icons/Plus05.png "[Add]") (Add) button in the *Assigned attribute sets* field is unlocked.   
 
 - *Key*   
     Enter an attribute key. The key is required for API access and must be system wide unique. An attribute key must fulfill the following criteria:
     - valid characters are **a-z** (upper and lower case), **0-9** and the underscore ( **_** )
     - the key must not start with a number
     - a double underscore ( **___** ) and a trailing underscore are forbidden
 
+[comment]: <> (UPDATE KEY -> Bitte aus Create attribute view abbauen!)
+
 - ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Multi-language*   
     Enable this toggle to create a multi-language attribute. Disable the toggle to create a single language attribute. By default, this toggle is disabled.
 
-- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Multi-channel*   
-    Enable this toggle to create a multi-channel attribute. Disable the toggle to create a single channel attribute. By default, this toggle is disabled.
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Multi-scope*   
+    Enable this toggle to create a multi-scope attribute. Disable the toggle to create a single scope attribute. By default, this toggle is disabled.
 
 - ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Active*   
     Enable this toggle to set the attribute status to active. Disable the toggle to set the status to inactive. By default, this toggle is enabled.
 
-- *Assigned sets*   
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Contains sensitive data*   
+    Enable this toggle if the potential attribute values may be sensitive. This toggle is only displayed after selecting certain data types in the *Data type* drop-down list. By default, this toggle is disabled.
```

Comment:
if the potential attribute -> wieso potential? verwirrend -> Enable this toggle if the attribute values are sensitive